src/Security/TwoFactor/GoogleAuthenticator.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Security\TwoFactor;
+
+use Scheb\TwoFactorBundle\Model\Google\TwoFactorInterface;
+use Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Google\GoogleAuthenticatorInterface;
+use Sonata\GoogleAuthenticator\GoogleAuthenticatorInterface as SonataGoogleAuthenticatorInterface;
+
+/**
+ * Copied from Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Google\GoogleAuthenticator to customize service name.
+ */
+class GoogleAuthenticator implements GoogleAuthenticatorInterface
+{
+    /**
+     * @var string
+     */
+    private $server;
+
+    /**
+     * @var SonataGoogleAuthenticatorInterface
+     */
+    private $authenticator;
+
+    /**
+     * @var string
+     */
+    private $issuer;
+
+    public function __construct(SonataGoogleAuthenticatorInterface $authenticator, $server, $issuer)
+    {
+        $this->authenticator = $authenticator;
+        $this->server = $server;
+        $this->issuer = $issuer;
+    }
+
+    public function checkCode(TwoFactorInterface $user, string $code): bool
+    {
+        // Strip any user added spaces
+        $code = str_replace(' ', '', $code);
```

Comment:
use trim()

Reply:
This code copied from original authenticator without changes. Anyway, what if spaces will be inside text?